src/main/kotlin/com/terraformation/backend/accelerator/db/VoteStore.kt

```diff
@@ -76,4 +89,22 @@ class VoteStore(
           .execute()
     }
   }
+
+  private fun getProjectCohortPhase(projectId: ProjectId): CohortPhase {
+    val record =
+        dslContext
+            .select(COHORTS.PHASE_ID)
+            .from(PROJECTS)
+            .join(PARTICIPANTS)
+            .on(PROJECTS.PARTICIPANT_ID.eq(PARTICIPANTS.ID))
+            .join(COHORTS)
+            .on(PARTICIPANTS.COHORT_ID.eq(COHORTS.ID))
+            .where(PROJECTS.ID.eq(projectId))
+            .fetchOne()
+
+    if (record == null) {
+      throw ProjectNotInCohortException(projectId)
+    }
+    return record.value1()!!
```

Comment:
This could be
```kotlin
return dslContext.select(COHORTS.PHASE_ID)
    // from+join+where as you have it in the PR
    .fetchOne(COHORTS.PHASE_ID)
    ?: throw ProjectNotInCohortException(projectId)
```